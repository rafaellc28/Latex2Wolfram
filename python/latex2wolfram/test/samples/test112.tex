% Test adjugate matrix
adj(\begin{bmatrix}
  8 & 7 & 7\\
  6 & 9 & 2\\
 -6 & 9 & -2\\
\end{bmatrix})
