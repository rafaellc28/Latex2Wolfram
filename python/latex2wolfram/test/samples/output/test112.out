adjugate({{8, 7, 7}, {6, 9, 2}, {-6, 9, -2}})
